packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyTable.tsx

```diff
@@ -39,14 +33,12 @@ export const CalculatedPropertyTable = ({
   onClickModify,
   isLoading,
   calculatedProperties,
-  refreshCalculatedProperties,
+  refresh,
 }: CalculatedPropertyTableProps) => {
-  const { getAccessToken, iModelId } = useGroupingMappingApiConfig();
   const mappingClient = useMappingClient();
-  const [showDeleteModal, setShowDeleteModal] = useState<CalculatedProperty | undefined>(undefined);
 
-  const calculatedPropertiesColumns = useMemo(
-    () => [
+  const columns = useCallback(
```

Comment:
Why did you change from `useMemo` to `useCallback` here? Also, this is not `columns` now but rather `columnsFactory`.

Reply:
I agree. `columnsFactory` is a lot clearer.